### ISSUE TITLE
Websockets support in ASP.NET Core: Update sample to 7.0 including controller updates with HttpConnect attribute

### ISSUE THREAD

**Issue by wadepickett** [dotnet/AspNetCore.Docs#26812]
The Websockets support in ASP.NET Core topic needs an update for a 7.0 app sample version and a fix for its controllers so that it is Http/2 compatible.

Per @BrennanConroy 

To fix controllers in the sample we would need something like:
```csharp
public class HttpConnectAttribute : HttpMethodAttribute
{
    private static readonly IEnumerable<string> _supportedMethods = new[] { "CONNECT" };

    public HttpConnectAttribute()
        : base(_supportedMethods)
    {
    }
}
---

Related to issue #26768 

#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a3b0cac1-adc7-434c-d867-6e28e39bdae7
* Version Independent ID: 762efeed-010e-f422-03d0-22d6681b2851
* Content: [WebSockets support in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/websockets?view=aspnetcore-6.0)
* Content Source: [aspnetcore/fundamentals/websockets.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/fundamentals/websockets.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-fundamentals**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by BrennanConroy** [comment]
I filed a product issue to see if we want to provide that type in the framework.
https://github.com/dotnet/aspnetcore/issues/43501

**Comment by wadepickett** [comment]
Fixed with #26768
